src/scripts.js

```diff
@@ -278,6 +278,9 @@ $(document).ready(function () {
              <p class="friends--steps"><b>${userName}</b>:</p>
              <p>${activity.returnFriendsStepCount()[0][userID]} steps</p>`;
     });
+    list += `<li class="friends_li">
+            <p class="friends--steps"><b>Highest step count:</b>:
+            ${findUserName(Number(activity.returnFriendsStepCount()[1]))}</p>`;
```

Comment:
There's quite a bit of logic in line 283 -- it's a little difficult to read. This is a scenario where I might break these things out into semantic variables so that I can more easily see what information I'm ending up with when all is said and done. e.g.:

```js
let something = activity.returnFriendsStepCount();
let somethingElse = Number(something[1]);
let userName = findUserName(somethingElse);
```

(Notice I'm having a hard time semantically naming these variables for you because I'm not 100% what each step of that code is giving back to me.)